Clean up optimized-oss.yaml

swolchok opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #8549
* #8597
* #8595

We don't need this now that we support log_softmax and gelu in OSS!

Differential Revision: [D69475020](https://our.internmc.facebook.com/intern/diff/D69475020/)

Commit:
Clean up optimized-oss.yaml

We don't need this now that we support log_softmax and gelu in OSS!

Differential Revision: [D69475020](https://our.internmc.facebook.com/intern/diff/D69475020/)

[ghstack-poisoned]

Commit:
Update on "Clean up optimized-oss.yaml"


We don't need this now that we support log_softmax and gelu in OSS!

Differential Revision: [D69475020](https://our.internmc.facebook.com/intern/diff/D69475020/)

[ghstack-poisoned]

swolchok commented:
wav2letter test is breaking. this change makes it use optimized log_softmax. not yet sure if this indicates an accuracy issue in optimized log_softmax or if the test tolerance just needs an update...

swolchok commented:
oh, the test value are WAY off:

```
eager output[0]: tensor([[-8.3237, -8.3237, -8.3236],
        [-8.3237, -8.3237, -8.3236],
        [-8.2994, -8.2991, -8.2993],
        ...,
        [-8.3237, -8.3237, -8.3236],
        [-8.3218, -8.3221, -8.3220],
        [-8.3113, -8.3117, -8.3116]])
executorch output[0]: tensor([[-8.3233, -8.3234, -8.3235],
        [ 0.9652,  0.9652, -8.3235],
        [ 0.9890,  0.9892, -8.2992],
        ...,
        [-8.3233, -8.3234, -8.3235],
        [-8.3215, -8.3218, -8.3219],
        [-8.3109, -8.3114, -8.3115]])
```

optimized log_softmax is clearly broken. need to expand tests for that op and figure this out

mergennachin commented:
> optimized log_softmax is clearly broken. need to expand tests for that op and figure this out

@manuelcandales

Commit:
Update on "Clean up optimized-oss.yaml"


We don't need this now that we support log_softmax and gelu in OSS!

Differential Revision: [D69475020](https://our.internmc.facebook.com/intern/diff/D69475020/)

[ghstack-poisoned]

swolchok commented:
there's some problem with supported_features for optimized kernels in OSS IIRC. will have to debug & fix

larryliu0820 reviewed (approved):
Thank you for doing this!

Commit:
Update on "Clean up optimized-oss.yaml"


We don't need this now that we support log_softmax and gelu in OSS!

Differential Revision: [D69475020](https://our.internmc.facebook.com/intern/diff/D69475020/)

[ghstack-poisoned]

swolchok commented:
> there's some problem with supported_features for optimized kernels in OSS IIRC. will have to debug & fix

OSS CMake setup was using the portable supported_features config for optimized. probably a copy/paste/forget-to-edit error. fixed.

swolchok committed:
Update on "Clean up optimized-oss.yaml"


We don't need this now that we support log_softmax and gelu in OSS!

Differential Revision: [D69475020](https://our.internmc.facebook.com/intern/diff/D69475020/)

[ghstack-poisoned]
